jira/client.py

```diff
@@ -847,14 +852,16 @@ def json_params() -> dict[str, Any]:
                         page_size,
                     )
                 page_start = (startAt or start_at_from_response or 0) + page_size
+                async_class = self._options["async_class"]
                 if (
                     async_class is not None
                     and not is_last
                     and (total is not None and len(items) < total)
                 ):
                     async_fetches = []
                     future_session = async_class(
-                        session=self._session, max_workers=async_workers
+                        session=self._session,
+                        max_workers=self._options["async_workers"],
```

Comment:
Just to preserve the original code I would suggest keep this as

```suggestion
                        max_workers=self._options.get("async_workers"),
```

Reply:
what value dose preserve the original code add in this case?